docs/apache-airflow/concepts.rst

```diff
@@ -448,7 +447,81 @@ Operators are only loaded by Airflow if they are assigned to a DAG.
+| ``poke`` (default) | The sensor is taking  | Use this mode if the  |
+|                    | up a worker slot for  | expected runtime of   |
+|                    | its whole execution   | the sensor is short   |
+|                    | time and sleeps       | or if a short poke    |
+|                    | between pokes.        | interval is required. |
+|                    |                       | Note that the sensor  |
+|                    |                       | will hold onto a      |
+|                    |                       | worker slot and a     |
+|                    |                       | pool slot for the     |
+|                    |                       | duration of the       |
+|                    |                       | sensor's runtime in   |
+|                    |                       | this mode.            |
++--------------------+-----------------------+-----------------------+
+| ``reschedule``     | The sensor task frees | Use this mode if the  |
+|                    | the worker slot when  | time before the       |
+|                    | the criteria is not   | criteria is met is    |
+|                    | yet met and it's      | expected to be quite  |
+|                    | rescheduled at a      | long. The poke        |
+|                    | later time.           | interval should be    |
+|                    |                       | more than one minute  |
+|                    |                       | to prevent too much   |
+|                    |                       | load on the           |
+|                    |                       | scheduler.            |
++--------------------+-----------------------+-----------------------+
+| ``smart sensor``   | smart sensor is a     | Use this mode if you  |
+|                    | service (run by a     | have a large amount   |
+|                    | builtin DAG) which    | of sensor tasks       |
+|                    | consolidate the       | running in your       |
+|                    | execution of sensors  | airflow cluster. This |
+|                    | in batches. Instead   | can largely reduce    |
+|                    | of holding a long     | airflow’s             |
+|                    | running process for   | infrastructure cost   |
+|                    | each sensor and       | and improve cluster   |
+|                    | poking periodically,  | stability - reduce    |
+|                    | a sensor will only    | meta database load.   |
+|                    | store poke context at |                       |
+|                    | ``sensor_instance``   |                       |
+|                    | table and then exits  |                       |
+|                    | with a 'sensing'      |                       |
+|                    | state.                |                       |
++--------------------+-----------------------+-----------------------+
```

Comment:
For this table, it's shown with a scrollbar horizontally and it's a bit long. Is there a way to wrap the text inside the cell instead?

Reply:
I'd probably suggest not making it a table then

Reply:
hmm... if that's the case, I will remove the table later